app/models/like.rb

```diff
@@ -0,0 +1,3 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] It would be better if the Like model has a method that `updates the likes counter for a post`